packages/vscode-extension/src/handlers.ts

```diff
@@ -370,6 +370,17 @@ export async function createNewProjectHandler(args?: any[]): Promise<Result<any,
   return result;
 }
 
+export async function createNewM365ProjectHandler(args?: any[]): Promise<Result<any, FxError>> {
+  ExtTelemetry.sendTelemetryEvent(TelemetryEvent.CreateProjectStart, getTriggerFromProperty(args));
```

Comment:
Use CreateM365ProjectStart event to distinguish the operations?

Reply:
Creating M365 app will have two entry points:
1. This new command.
2. The original creating command (just be informed from PM today. I will be added later).
I am not sure how to distinguish creating M365 apps from creating the original Teams app. What is your idea?

Reply:
If user trigger from the new command, we can easily distinguish since different handlers are triggered. If user trigger from original creating command, I think we can just distinguish from the question model telemetry.

Reply:
Added is-m365 property to creating project event when creating M365 app. Thanks~